module/VuFind/src/VuFind/Db/Table/ResourceTags.php

```diff
@@ -382,37 +384,7 @@ public function destroyListLinks($list, $user, $tag = null)
                 }
             }
         };
-        $this->processDestroyLinks($callback);
-    }
-
-    /**
-     * Process link rows marked to be destroyed.
-     *
-     * @param Object $callback Callback function for selecting deleted rows.
-     *
-     * @return void
-     */
-    protected function processDestroyLinks($callback)
```

Comment:
This processDestroyLinks logic was intended to clean up orphaned tag rows after tags are unlinked. However, it's just begging for race conditions and other problems; I thought it best to drop it entirely rather than try to port it forward. If we want to clean up orphaned tags, it's probably better to do it via the tag admin or a command line utility. Given how low tag utilization has been historically, I don't think accumulating orphaned tags is a huge threat in any case.